src/reynir_correct/errfinder.py

```diff
@@ -697,17 +700,46 @@ def find_verb_subject(tnode: SimpleTree) -> Optional[SimpleTree]:
                 pass
         if subj is None:
             # If not found there, look within the
-            # enclosing IP (inflected phrase) node, if any
+            # enclosing IP (inflectional phrase) node, if any
             p = tnode.enclosing_tag("IP")
             if p is not None:
-                # Found the inflected phrase:
+                # Found the inflectional phrase:
                 # find the NP-SUBJ node, if any
                 try:
                     subj = p.NP_SUBJ
                 except AttributeError:
                     pass
         return subj
 
+    @staticmethod
+    def find_verb_direct_object(tnode: SimpleTree) -> Optional[SimpleTree]:
+        """ Starting with a verb terminal node, attempt to find the 
+        verb's direct object noun phrase """
+        obj = None
```

Comment:
I would have thought you needed a type annotation here: `obj: Optional[SimpleTree] = None`

Reply:
Copied this mostly from find_verb_subject(); fixed in both places